Buildings/Fluid/Boilers/Data/Generic.mo

```diff
@@ -0,0 +1,53 @@
+within Buildings.Fluid.Boilers.Data;
+record Generic "Generic data record for boiler performance"
+  extends Modelica.Icons.Record;
+
+  parameter Modelica.SIunits.Efficiency effCur[:,:]=
+    [0, 1; 1, 1]
+    "Efficiency curves as a table: First row = inlet temp(K), First column = firing rates or PLR";
+  final parameter Modelica.SIunits.Efficiency eta_nominal=
+    Buildings.Utilities.Math.Functions.smoothInterpolation(
+      x=TIn_nominal,
+      xSup=effCur[1,2:end],
+      ySup=effCur[end,2:end])
+    "Efficiency at TIn_nominal";
+  parameter Modelica.SIunits.Temperature TIn_nominal = 323.15
+    "Nominal inlet temperature for efficiency calculations";
+
+  parameter Buildings.Fluid.Data.Fuels.Generic fue "Fuel type"
+   annotation (choicesAllMatching = true);
+
+  parameter Modelica.SIunits.Power Q_flow_nominal "Nominal heating power";
+  parameter Modelica.SIunits.ThermalConductance UA=0.05*Q_flow_nominal/30
+    "Overall UA value";
+  parameter Modelica.SIunits.Volume VWat = 1.5E-6*Q_flow_nominal
+    "Water volume of boiler";
+  parameter Modelica.SIunits.Mass mDry =   1.5E-3*Q_flow_nominal
```

Comment:
remove the empty spaces, so it will be `mDry = 1.5E-3*Q_flow_nominal`.

Reply:
Removed in 7535616.